### ISSUE TITLE
Creation of RediSearch index fails due to inconsistent error message

### ISSUE THREAD

**Issue by andrewkdouglas** [getmetal/motorhead#68]
When first started Motorhead fails with the error `RediSearch index error: Unknown: index name`. 

This seems to be cause by an inconsistent error message from RediSearch. Motörhead is expecting `Unknown: Index name`, RediSearch is returning `Unknown: index name` so initialisation of the index fails.

Seems to be related to: https://github.com/RediSearch/RediSearch/issues/896

**Platform:** Mac M2
**Error message:**
```
Attaching to motorhead-motorhead-1, motorhead-redis-1
motorhead-redis-1      | 8:C 20 Aug 2023 15:24:28.176 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
motorhead-redis-1      | 8:C 20 Aug 2023 15:24:28.176 * Redis version=7.2.0, bits=64, commit=00000000, modified=0, pid=8, just started
motorhead-redis-1      | 8:C 20 Aug 2023 15:24:28.176 * Configuration loaded
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.177 * monotonic clock: POSIX clock_gettime
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.177 * Running mode=standalone, port=6379.
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.178 * <search> Redis version found by RedisSearch : 7.2.0 - oss
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.178 * <search> RediSearch version 2.8.4 (Git=HEAD-7797d39d)
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.178 * <search> Low level api version 1 initialized successfully
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.178 * <search> concurrent writes: OFF, gc: ON, prefix min length: 2, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  10000, search pool size: 20, index pool size: 8,
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.179 * <search> Initialized thread pools!
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.179 * <search> Enabled role change notification
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.179 * Module 'search' loaded from /opt/redis-stack/lib/redisearch.so
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> RedisTimeSeries version 11004, git_sha=c10178d33caf11177e12ca575092b072d81a5cd2
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> Redis version found by RedisTimeSeries : 7.2.0 - oss
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> loaded default CHUNK_SIZE_BYTES policy: 4096
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> loaded server DUPLICATE_POLICY: block
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> Setting default series ENCODING to: compressed
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.180 * <timeseries> Detected redis oss
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.185 * Module 'timeseries' loaded from /opt/redis-stack/lib/redistimeseries.so
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Created new data type 'ReJSON-RL'
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> version: 20606 git sha: unknown branch: unknown
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Exported RedisJSON_V1 API
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Exported RedisJSON_V2 API
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Exported RedisJSON_V3 API
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Exported RedisJSON_V4 API
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <ReJSON> Enabled diskless replication
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * Module 'ReJSON' loaded from /opt/redis-stack/lib/rejson.so
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <search> Acquired RedisJSON_V4 API
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <bf> RedisBloom version 2.6.3 (Git=unknown)
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * Module 'bf' loaded from /opt/redis-stack/lib/redisbloom.so
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.186 * <redisgears_2> Created new data type 'GearsType'
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.187 * <redisgears_2> Detected redis oss
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.187 # <redisgears_2> could not initialize RedisAI_InitError
motorhead-redis-1      |
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.187 * <redisgears_2> Failed loading RedisAI API.
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.187 * <redisgears_2> RedisGears v2.0.11, sha='0aa55951836750ceabd9733decb200f8a5e7bac3', build_type='release', built_for='Linux-ubuntu22.04.arm64v8'.
motorhead-redis-1      |
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.188 * <redisgears_2> Registered backend: js.
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.188 * Module 'redisgears_2' loaded from /opt/redis-stack/lib/redisgears.so
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.188 * Server initialized
motorhead-redis-1      | 8:M 20 Aug 2023 15:24:28.188 * Ready to accept connections tcp
motorhead-motorhead-1  | [2023-08-20T15:24:28Z INFO  motorhead] Starting Motorhead 🤘
motorhead-motorhead-1  | RediSearch index error: Unknown: index name
motorhead-motorhead-1 exited with code 1
```

**Comment by Czechh** [comment]
Thank you!

**Comment by andrewkdouglas** [comment]
Thanks for merging 👍 

Is it possible for this to be released as v2.0.3, please?

**Comment by Czechh** [comment]
Having some issues with the build, looking into it.
